.github/workflows/server-ci.yml

```
@@ -411,3 +411,93 @@ jobs:
+      WEBHOOK_URL_FLAKY_TEST: ${{ secrets.WEBHOOK_URL_FLAKY_TEST }}
+      WEBHOOK_AUTH_TOKEN_FLAKY_TEST: ${{ secrets.WEBHOOK_AUTH_TOKEN_FLAKY_TEST }}
+    with:
+      source-run-id: ${{ github.run_id }}
+      source-event: ${{ github.event_name }}
+      head-commit-id: ${{ github.event.pull_request.head.sha || github.sha }}
+      head-repository: ${{ github.event.pull_request.head.repo.full_name || github.repository }}
+      workflow-url: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+
+  ci-artifacts:
+    name: Server CI Artifacts
+    if: >-
```

actionlint (1.7.12)

[error] number of arguments is wrong. function "success() -> bool" takes 0 parameters but 1 arguments are given

(expression)